index.html

```diff
@@ -1879,6 +1879,88 @@ <h3>Versioning Cryptography Suites</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Layer Cryptographic Options Appropriately</h3>
```

Comment:
I think this is not just about Application Developers, and I agree that "appropriately" can be problematic. Hence —
```suggestion
        <h3>Considerations when Layering Cryptographic Options</h3>
```

Reply:
The titles need to be 3 words long or they wrap on most devices, we want something pithy. The title you provided is accurate, but might not result in something "sticky".

Suggesting "Prioritize Application Developers" as a "Security Consideration".